tests/domain/test_amino_acid_d.py

```diff
@@ -2,12 +2,11 @@
 
```

Comment:
For all tests subfolders, I renamed each .py file using the beginning letter of the subfolder and putting it at the end. It's clearer than using __init__.py files, since tests folder is not a package.

Reply:
OK, I guess the tests will still work this way.